doc/source/generate_module_rst.py

```diff
@@ -202,10 +202,7 @@ def write_to_rst_file(node_name: str, rst_content: List[str]) -> None:
 
     level_underline = RST_LEVEL_SYMBOLS[0] * len(node_name)
     rst_file = SPHINX_FILE_STRING.format(
-        title=node_name,
```

Comment:
It looks like this has been reformated with an older version of `black`. The 'black' version was recently bumped to 20.8b. Which version was used for this?

Reply:
yes, you are correct.

I reinstalled my venv and re-ran (the latest changes should reflect this)